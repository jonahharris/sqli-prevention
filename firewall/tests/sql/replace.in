>>> Sada testovacich SQL dotazu: replace
REPLACE t1 VALUE (1, 2);
REPLACE LOW_PRIORITY INTO t1 VALUE (1, 2);
REPLACE DELAYED INTO t1 VALUE (1, 2);
REPLACE t1 SET a = 1, b = 2;
REPLACE LOW_PRIORITY INTO t1 SELECT * FROM t2;
REPLACE INTO test VALUES (1, 'New', '2014-08-20 18:47:42');
REPLACE INTO test2 VALUES (1, 'Old', '2014-08-20 18:47:00');
