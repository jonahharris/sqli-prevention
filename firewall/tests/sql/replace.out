>>> Sada testovacich SQL dotazu: replace
NUMBER 1;NUMBER 2;VALUES 2;TABLE t1;REPLACE VALS 0 1;STMT;1;0;
NUMBER 1;NUMBER 2;VALUES 2;TABLE t1;REPLACE VALS 1 1;STMT;1;0;
NUMBER 1;NUMBER 2;VALUES 2;TABLE t1;REPLACE VALS 2 1;STMT;1;0;
NUMBER 1;ASSIGN a;NUMBER 2;ASSIGN b;TABLE t1;REPLACE ASGN 0 2;STMT;1;0;
SELECTALL;TABLE t2;SELECT 0 1 1;TABLE t1;REPLACE SELECT 1;STMT;1;0;
NUMBER 1;STRING '***';STRING '***';VALUES 3;TABLE test;REPLACE VALS 0 1;STMT;1;0;
NUMBER 1;STRING '***';STRING '***';VALUES 3;TABLE test2;REPLACE VALS 0 1;STMT;1;0;
